framework/src/postprocessors/SideAverageFunctorPostprocessor.C

```diff
@@ -0,0 +1,74 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "SideAverageFunctorPostprocessor.h"
+
+registerMooseObject("MooseApp", SideAverageFunctorPostprocessor);
+registerMooseObject("MooseApp", ADSideAverageFunctorPostprocessor);
```

Comment:
Since the creation of the `NonADFunctorInterface`, which `Postprocessor` inherits from, we no longer need to template postprocessor classes that retrieve functors. This is because we don't care about dropping derivatives so behind the scenes we wrap the possibly-AD functors in a wrapper class which applies `MetaPhysicL::raw_value` to every invocation of the wrapped functor evaluation. So you can remove the templating and keep only the `SideAverageFunctorPostprocessor`. Optionally you can also remove the templating of the base class. If you don't, then just make sure to inherit from `SideIntegralFunctorPostprocessorTempl<false>`

Reply:
Thanks. Defaulted `SideAverageFunctorPostprocessor ` to non-AD. I am not changing `SideIntegralFunctorPostprocessorTempl` since their are a bunch of inherited parameters on the ad template that you may want to take another look at later on.